thavalon-webapp/src/components/gameComponents/roleInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType, OutboundMessageType } from "../../utils/GameSocket";
+import "../../styles/gameStyles/roleInformation.scss";
+
+interface RoleInfo {
+    abilities: string,
+    assassinatable: boolean,
+    description: string,
+    isAssassin: boolean,
+    otherInfo: string,
+    priorityTarget: string | undefined,
+    role: string,
+    seenPlayers: string[],
+    team: string,
+    teamMembers: string[],
+}
+
+interface Snapshot {
+    roleInfo: RoleInfo,
+    missions: string[],
```

Comment:
This is an array of structs - each has the proposals and voting and stuff for that mission

Reply:
changed both missions and log to any[] -> We can determine actual values and create consts as part of PR where we actually use these